docs/src/pages/demos/selection-controls/CheckboxesGroup.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { createStyles, Theme, withStyles, WithStyles } from '@material-ui/core/styles';
+import FormLabel from '@material-ui/core/FormLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import Checkbox from '@material-ui/core/Checkbox';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      display: 'flex',
+    },
+    formControl: {
+      margin: theme.spacing(3),
+    },
+  });
+
+export interface Props extends WithStyles<typeof styles> {}
+
+function CheckboxesGroup(props: Props) {
+  const { classes } = props;
+  const [state, setState] = React.useState({
+    gilad: true,
+    jason: false,
+    antoine: false,
+  });
+
+  const handleChange = (name: string) => (event: React.ChangeEvent<HTMLInputElement>) => {
+    setState({ ...state, [name]: event.target.checked });
+  };
+
+  const { gilad, jason, antoine } = state;
+  const error = [gilad, jason, antoine].filter(v => v).length !== 2;
+
+  return (
+    <div className={classes.root}>
+      <FormControl component={'fieldset' as 'div'} className={classes.formControl}>
+        <FormLabel component={'legend' as 'div'}>Assign responsibility</FormLabel>
```

Comment:
What `'fieldset' as 'div'` means is to lie to typescript that we actually use the default component. For `FormControl` this is `div` for `FormLabel` it would be `label`.

I guess `legend` is rejected? If so we need to cast it the the default component.